WebApplicationDemo/Startup.cs

```diff
@@ -2,41 +2,43 @@
 using System.Collections.Generic;
 using System.IdentityModel.Tokens.Jwt;
 using System.IO;
-using System.Linq;
-using System.Threading;
-using System.Threading.Tasks;
+
 using Ark.Tools.AspNetCore.HealthChecks;
 using Ark.Tools.AspNetCore.Startup;
 using Ark.Tools.AspNetCore.Swashbuckle;
 
+using Asp.Versioning;
+
 using Microsoft.AspNetCore.Authentication.JwtBearer;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Builder;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.HttpsPolicy;
-using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.OData;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Diagnostics.HealthChecks;
 using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.Logging;
 using Microsoft.Identity.Web;
 using Microsoft.OpenApi.Models;
-using Swashbuckle.AspNetCore.Swagger;
+
 using Swashbuckle.AspNetCore.SwaggerUI;
+
 using WebApplicationDemo.Application.Host;
 using WebApplicationDemo.Dto;
 
 namespace WebApplicationDemo
 {
-	public class Startup : ArkStartupWebApi
+    public class Startup : ArkStartupWebApi
 	{
 		public Startup(IConfiguration configuration, IHostEnvironment env)
 			: base(configuration, env, false)
 		{
         }
 
-		public override IEnumerable<ApiVersion> Versions => new[] { new ApiVersion(1, 0) };
+		public override IEnumerable<ApiVersion> Versions => new[] 
+        { 
+            new ApiVersion(1, 0),
```

Comment:
use the ones defined in [WebApplicationDemo/Configuration/ApiVersions.cs](https://github.com/ARKlab/Ark.Tools/pull/51/files#diff-e5c21edca121984d215b2c35fa8f29732415bec6e37b215eaf52bf02e0f5c24c)

actually, would be best if in the ApiVersion.cs is introduced a new constant `ApiVersion[] All`  which is referenced here.